changelog/15904.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
I'd probably remove the changelog entry based on the discussion on the earlier thread.

Reply:
No problem, I was thinking the same thing but then succumbed after searching for other doc related changelog entries and saw there were >1 (2 maybe). I'll remove it. 👍🏼